File: MasterLogicaProgramacion/EjerciciosPython/Ej9.py
# Dados dos arrays, devolver un array con los números comunes entre ambos

def devolverComunes(arr1, arr2):
    arr1NoDuplicados = []
    [arr1NoDuplicados.append(x) for x in arr1 if x not in arr1NoDuplicados]

    return [a for a in arr1NoDuplicados if a in arr2]

print(devolverComunes([7,8,9,7,5], [4,5,6,7]))

File: Cursos/MasterLogicaProgramacion/EjerciciosJavascript/Ej9.js
/*
    Dados dos arrays, devolver un array con los números comunes entre ambos
*/

function devolverComunes(arr1, arr2) {
    return arr1.filter((item, index) => arr1.indexOf(item) === index).filter(x => arr2.includes(x));
}

console.log(devolverComunes([7,8,9,7,5], [4,5,6,7]));